stable/dokuwiki/meta.yaml

```diff
@@ -1,6 +1,6 @@
 name: "dokuwiki"
 description: "DokuWiki is a standards-compliant, simple to use wiki optimized for creating documentation. It is targeted at developer teams, workgroups, and small companies. All data is stored in plain text files, so no database is required."
 categories:
-  - "demo"
+  - "Demo Apps"
```

Comment:
Please use `category.name`, not `category.display_name`

Reply:
I will do it now